Happy-Hour/src/main/java/com/happyhour/HappyHour/models/dto/LoginFormDTO.java

```diff
@@ -0,0 +1,35 @@
+package com.happyhour.HappyHour.models.dto;
+
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
+public class LoginFormDTO {
+
+    @NotNull
+    @NotBlank
+    @Size(min = 3, max = 20, message = "Invalid username. Must be between 3 and 30 characters.")
```

Comment:
The size parameters indicate min=3 and max=20, but the message says a max of 30 characters.  Also, the Owner.java class says 3 to 50.